package.json

```diff
@@ -144,11 +144,11 @@
     "cors": "^2.8.5",
     "css-loader": "^1.0.0",
     "cssnano": "^4.1.10",
+    "cy-tr-reporter": "https://github.com/tlei123/cy-tr-reporter",
```

Comment:
@tlei123 is there a way to call this as a module? because people will have issues on their pipeline if the pipeline has no access to github. We had that issue with jsPDF having gitHub on it because we use GitLab for our repos and this was the fix - https://github.com/mbrn/material-table/issues/2109#issuecomment-655011498